src/pages/users.vue

```
@@ -0,0 +1,156 @@
+                </v-card-actions>
+            </v-card>
+        </v-dialog>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue';
+import Search from '@/components/Search.vue';
+import { searchUsers, updateUser, deleteUser } from '@/services/userService';
+import type { User } from '@/models/user';
+import { formatDate } from '@/utils/formatDate';
```

ESLint

[error] 'formatDate' is defined but never used.

(@typescript-eslint/no-unused-vars)